msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md

```diff
@@ -114,7 +114,7 @@ Messaging extension works as expected when a user is in an in-meeting view. The
 
 #### In-meeting dialog box
 
-The in-meeting dialog box is used to engage participants during the meeting and collect information or feedback during the meeting. Use the [`NotificationSignal`](API-references.md#notificationsignal-api) API to trigger a bubble notification. As part of the notification request payload, include the URL where the content to be shown is hosted.
+The in-meeting dialog box is used to engage participants during the meeting and collect information or feedback during the meeting. [Send notification signal API](API-references.md#send-notification-signal-api) API to trigger a bubble notification. As part of the notification request payload, include the URL where the content to be shown is hosted.
```

Comment:
Use the [`SendNotificationSignal`](API-references.md#send-notification-signal-api)